### PR TITLE
Search: Add support for datetime range queries

### PR DESCRIPTION
**Pull request by olovy**

### CODE REVIEW

**Comment by olovy** [comment]
Integration tests: https://github.com/libris/lxl_api_tests/pull/11

File: rest/API.md [hunk 1]
```diff
@@ -173,23 +197,46 @@ $ curl -XGET -H "Accept: application/ld+json" \
...
-Find posts containing 'tove' and linking to saogf/Romaner.
+Published in the 1760s.
 ```
 $ curl -XGET -H "Accept: application/ld+json" \
-    'https://libris-qa.kb.se/find?q=tove&o=https://id.kb.se/term/saogf/Romaner&_limit=3'
+    'https://libris-qa.kb.se/find.jsonld?min-publication.year=1760&maxEx-publication.year=1770&_limit=5'
+...
+```
+
+#### Exempel
```
Comment (lrosenstrom) [inline rest/API.md]: A bit of swedish crept in here -->

File: rest/API.md [hunk 2]
```diff
@@ -173,23 +197,46 @@ $ curl -XGET -H "Accept: application/ld+json" \
...
+    -d instanceOf.@type=NotatedMusic \
+    -d min-publication.year=1930 \
+    -d max-publication.year=1939 \
+    -d min-publication.year=1950 \
+    -d max-publication.year=1959 \
+    -d _limit=5
+...
+```
+
+#### Exempel
```
Comment (lrosenstrom) [inline rest/API.md]: -->

**Comment by olovy** [comment]
> Very nice!

:pray:

**Pull request merged**